LightGBM/model.py

```diff
@@ -0,0 +1,40 @@
+import lightgbm as lgb
+
+# All available parameters can be found here:  
+# https://lightgbm.readthedocs.io/en/latest/Parameters.html
+default_params = {
+                  "objective" : "poisson",
```

Comment:
Put each item on one indentation and the last curly brace at the start of a line.

default_params = {
    "objective": "poisson"
}